addons/config/testcases/cni.tanzu.vmware.com/v1alpha1/antreaconfig/default.yaml

```diff
@@ -0,0 +1,2 @@
+#@data/values
+---
```

Comment:
**question**: Is there a way to not specify  testcase if its empty? i.e. if no data values then dont have default.yaml

Reply:
Not that I am aware. It might requires a bit ad-hoc logic in the `test.sh`. If you have cleaner way to handle this plz let me know

Reply:
I guess it could be handled similar to 

```
        path=${PACKAGE_REPOSITORY}/${name}
	mkdir -p "$path/$version/bundle/.imgpkg"
	yttCmd="${TOOLS_BIN_DIR}/ytt --ignore-unknown-comments -f $path/$version/bundle/config/"
	if [ -d "$path/$version/sample-values" ]
	then
		for sample_value_file_path in "$path/$version/sample-values"/*; do
			yttCmd="${yttCmd} -f ${sample_value_file_path}"
			# Only includes the first sample value file
			break
		done
	fi
	${yttCmd} | "${TOOLS_BIN_DIR}"/kbld -f - -f kbld-config.yml --imgpkg-lock-output "$path/$version/bundle/.imgpkg/images.yml" > /dev/null

```